js/global.js

```diff
@@ -657,7 +657,7 @@ jQuery(document).ready(function($) {
                 url: gdn.url(url),
                 data: {DeliveryType: 'VIEW'},
                 success: function(data) {
-                    $elem.htmlTrigger(data);
+                    $elem.html($.parseHTML(data + '')).trigger('contentLoad');
```

Comment:
Just a little note that `$.parseHTML` is not required here since `html()` will parse the html.